packages/backend/src/helpers/flow-templates.ts

```diff
@@ -35,6 +46,68 @@ function validateAppAndEventKey(step: TemplateStep, templateName: string) {
   }
 }
 
+// This replaces the keys of columns which are names to the column ids for row data
+// e.g. { Rating: 5 } --> { column_id: 5 }
+function replaceRowData(
```

Comment:
More accurate function naming pls
```suggestion
function replaceColumnNameWithId(
```